README.md

```diff
@@ -140,6 +140,35 @@ float
 
 </table>
 
+## Coding rules
+
+We define a list of coding rules for the OpenCL-code so that facilitate using it from multiple frameworks.
+
+### Source and destination images/buffers.
+
+There are many image processing filters which just take one source image and produce one destination image. Those should be simply called `src` and `dst`. In case multiple source and/or destination images appear in the kernel, those should be called `src0`, `src1`, ...
```

Comment:
One question would be about the priority of the tags. For now we have `src` > `dst` > `parameters`. I am fine with it though, from what I saw, all kernel have a 'dst' but some do not have 'src'. It would be logical to then do 'dst' > 'src' > 'parameters' but that would go against CLIJ standard. Your call :smiley:

Reply:
I suggest, we keep src, dst, parameters. It's also in CLIJ publications like that and doesn't hurt much in daily practice.